scripts/patch-pydoctor.sh

```diff
@@ -12,12 +12,21 @@ set -e
 ROOT_FOLDER=$1
 PATCH_FOLDER=$(realpath $2)
 ${ROOT_FOLDER}/.venv/bin/pip uninstall -y pydoctor
-${ROOT_FOLDER}/.venv/bin/pip install pydoctor==21.2.2
+#${ROOT_FOLDER}/.venv/bin/pip install pydoctor==21.2.2
+${ROOT_FOLDER}/.venv/bin/pip install pydoctor==21.12.0
 PYDOCTOR_DIR=`.venv/bin/python -c 'import os,pydoctor;print(os.path.dirname(pydoctor.__file__))'`
 
 cd "${PYDOCTOR_DIR}"
-# patch is confirmed to work with pydoctor 21.2.2
-patch -r deleteme.rej -N -p2 <${PATCH_FOLDER}/pydoctor-21.2.2.patch 2>/dev/null
-rm -f deleteme.rej
-
+## patch is confirmed to work with pydoctor 21.2.2
```

Comment:
You can remove the commented parts here if we have decided to migrate to PyDoctor 21.12.

Reply:
yep